### PR TITLE
(tortoisegit) migrated from dtgm

### PR DESCRIPTION
**Pull request by pascalberger**
* Package source: https://github.com/dtgm/chocolatey-packages/pull/265
* [No open issues](https://github.com/dtgm/chocolatey-packages/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20tortoisegit)

## Changes
Compared to the version from dtgmI changed the following stuff:
    * Added AU script
    * Update NuSpec metadata to be compliant with core team guidelines
    * Change to embedded package

### CODE REVIEW

**Comment by majkinetor** [comment]
Epic. Dtgm repo should be  scavanged with multuple packages.

I have very nice script to contribute that makes tortoisegit 100% operational from console.

**Comment by gep13** [comment]
@majkinetor @pascalberger  I have added the chocolatey user to this package, so as soon as the PR to dtgm repo is merged, this one is good to go 👍

File: automatic/tortoisegit/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  $filePath64
+} else { 
+  Write-Host "Installing 32 bit version"
+  $filePath32
+}
+
+$packageArgs = @{
+    PackageName = 'tortoisegit'
+    FileType = 'msi'
+    SoftwareName = 'Git version*'
```
Comment (AdmiringWorm) [inline automatic/tortoisegit/tools/chocolateyInstall.ps1]: is this really the software name?
looks kinda wrong to me

File: automatic/tortoisegit/tortoisegit.nuspec [hunk 2]
```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <metadata>
+    <id>tortoisegit</id>
+    <title>TortoiseGit</title>
+    <version>2.3.0.0</version>
+    <authors>TortoiseGit and contributors</authors>
+    <owners>chocolatey</owners>
```
Comment (AdmiringWorm) [inline automatic/tortoisegit/tortoisegit.nuspec]: missing original owner/maintainer

File: automatic/tortoisegit/tools/chocolateyInstall.ps1 (hunk 1 above)
Comment (pascalberger) [inline automatic/tortoisegit/tools/chocolateyInstall.ps1]: Fixed

File: automatic/tortoisegit/tortoisegit.nuspec (hunk 2 above)
Comment (pascalberger) [inline automatic/tortoisegit/tortoisegit.nuspec]: Fixed

**Comment by AdmiringWorm** [comment]
hmm, looks like 32bit installation fails in the 64bit test environment.
it fails with exit code 1603.

also on another note, please don't rebase your commits unless there is a conflict or you're asked.
it makes it harder to review what you changed, and requires to look through all of the files anew.
thus makes the reviewing process longer than needed.

**Comment by AdmiringWorm** [comment]
looks like 32bit installation in a 64bit environment isn't supported, maybe put a message in the install script if a user tries to do that?
and possibly a note in the description as well.

**Comment by pascalberger** [comment]
@AdmiringWorm Also simplified the AU update script a little bit, based on the review comments you left at the autoit package

**Comment by AdmiringWorm** [comment]
@pascalberger nice, I like it.
can't believe I missed it, I must be going blind 😞

**Comment by AdmiringWorm** [comment]
@pascalberger I had to push some changes to this PR, Get-RemoteFiles didn't behave as I expected it to.
also fixed some indentation issues.

**Comment by pascalberger** [comment]
@AdmiringWorm TortoiseGit already has a 4-part version number. I therefore suggest to wait for next update of TortoiseGit for a push.

**Comment by AdmiringWorm** [comment]
@pascalberger 
Or you can make it use the script I recently added to the repository:
https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/scripts/Get-Padded-Version.ps1

See the flashplayer packages as how I made those use it.

**Comment by pascalberger** [comment]
@AdmiringWorm Yeah, thought about it :) But since they announced to release 2.4.0 soon IMHO there's no need to push now.

**Comment by AdmiringWorm** [comment]
@pascalberger Okay, didn't know that. Let's just play the waiting game then.

**Pull request merged**